tests/Unleash.Tests/DefaultUnleashTests.cs

```diff
@@ -132,7 +132,13 @@ public static IUnleash CreateUnleash(string name, ToggleCollection state)
             var httpClient = new HttpClient(fakeHttpMessageHandler) { BaseAddress = new Uri("http://localhost") };
             var fakeScheduler = A.Fake<IUnleashScheduledTaskManager>();
             var fakeFileSystem = new MockFileSystem();
-            var toggleState = Newtonsoft.Json.JsonConvert.SerializeObject(state);
+            var toggleState = Newtonsoft.Json.JsonConvert.SerializeObject(state, new Newtonsoft.Json.JsonSerializerSettings
```

Comment:
For other reviewers: this is a very clever little trick, compliments of @nunogois.

The existing state in the tests is now serialised and passed like that. This allows us to preserve the bulk of the tests with minimal changes and have confidence that we don't do too much surgery on the tests while we're also changing the code

The serialiser change here is to get .NET to send camelCased Json rather than PascalCased JSON, which Yggdrasil doesn'tt like

Reply:
If making a major (semver) change for this release, why not go to System.Text.Json, which would then allow for further non-breaking work to adopt JSON source generators.

Reply:
Yep. we chatted, it's a good idea, we're gonna go all in on System.Text.Json. Not in this PR though, this one's already too big